Testing/functional/tests/cypress/integration/ui/services/view.js

```diff
@@ -19,10 +19,6 @@ describe("Authenticated Services View", () => {
         cy.visit(servicesTestsConstants.servicesUrl);
     });
 
-    afterEach(() => {
-        Cypress.session.clearAllSavedSessions();
-    });
-
     it("The page should load correctly", () => {
```

Comment:
Should there be a test where services enabled = false?  What will the VUE router do when services enabled = false?

Reply:
I'll add one it currently goes to the unauthorized page, which is the same behaviour as dependents.

Reply:
![image](https://user-images.githubusercontent.com/19548348/224440965-a299f5ad-2ad6-43fb-9594-c62d52fa9c33.png)

Added the test as suggested